### ISSUE TITLE
espeak for android  announces نیمفاصله while should not.

### ISSUE THREAD

**Issue by zahra21** [espeak-ng/espeak-ng#1207]
hello.
espeak for android announces نیمفاصله while i wish that does not announce this at all.
espeak for windows thanks God has my desired behaviour and does not announce it unless i set in for example nvda dictionary to says it!

**Comment by MH140000** [comment]
hi. yes. I have this problem. please fix this.
